Changes.md

```diff
@@ -1,5 +1,15 @@
 # Medic Mobile Release Notes
 
+## 2.8.6
+
+### TBD
+
+#### Continuous Integration
+
+- Automatically update to master branches of `api` and `sentinel` submodules
+  before running CI in Travis. [#3621](https://github.com/medic/medic-webapp/pull/3621)
+- Fixed kanso packages that inadvertantly cached data. Issue: #3648
+
```

Comment:
Ok I don't mind either way.  But I don't think the audience  is only app consumers.